js/util/struct_array.js

```diff
@@ -204,15 +261,20 @@ const structArrayTypeCache = {};
  *
  * @private
  */
-function createStructArrayType(options) {
+
+function createStructArrayType(options: {|
+  members: Array<{type: ViewType, name: string, components?:? number}>,
```

Comment:
Does there need to be two questionmarks, or is it enough to mark `components` optional?